src/ICS26Router.sol

```diff
@@ -256,25 +269,27 @@ contract ICS26Router is IICS26Router, IICS26RouterErrors, Ownable, ReentrancyGua
             return noopOnCorrectReason(reason, IICS24HostErrors.IBCPacketCommitmentNotFound.selector);
         }
 
-        getIBCApp(payload.sourcePort).onTimeoutPacket(
+        try getIBCApp(payload.sourcePort).onTimeoutPacket(
             IIBCAppCallbacks.OnTimeoutPacketCallback({
                 sourceChannel: msg_.packet.sourceChannel,
                 destinationChannel: msg_.packet.destChannel,
                 sequence: msg_.packet.sequence,
                 payload: payload,
                 relayer: _msgSender()
             })
-        );
+        ) {} catch {
+            // no-op
+        }
```

Comment:
Again, we need the timeout callback to succeed to unescrow funds

Reply:
Ditto: What if the timeout callback can never succeed?

Reply:
Then I think the packet should just be stuck?